Common/include/option_structure.hpp

```diff
@@ -2765,120 +2863,128 @@ class COptionRiemann : public COptionBase {
 };
 
 template <class Tenum>
-class COptionNRBC : public COptionRiemann<Tenum> {
+class COptionNRBC : public COptionBase{
+
+  map<string, Tenum> m;
+  unsigned short & size;
+  string * & marker;
+  unsigned short* & field; // Reference to the fieldname
+  string name; // identifier for the option
+  su2double * & var1;
+  su2double * & var2;
+  su2double ** & flowdir;
+  su2double * & relfac1;
+  su2double * & relfac2;
 
 public:
-	  COptionNRBC(string option_field_name, unsigned short & nMarker_NRBC, string* & Marker_NRBC, unsigned short* & option_field,
```

Comment:
NR stands for Non-reflecting? isn't it? Maybe we should be a little more explicit and use something like COptionNonReflectingBC